lib/src/primer_progress_bar.dart

```diff
@@ -88,16 +91,17 @@ class PrimerProgressBar extends StatelessWidget {
           style: barStyle,
           maxTotalValue: maxTotalValue,
         ),
-        SegmentedBarLegend(
-          style: SegmentedBarLegendStyle(
-            maxLines: null,
-            spacing: legendStyle.spacing,
-            runSpacing: legendStyle.runSpacing,
-            padding: legendStyle.padding,
+        if (showLegend)
+          SegmentedBarLegend(
```

Comment:
I think it's better to return the `SegmentedBar` itself if `showLegend` is false, rather than returning a `Column` that has only one widget.

Reply:
Done, could you check it out